bostardecksweb/urls.py

```diff
@@ -25,6 +25,7 @@
     path('api/v1/', include('api_v1.urls')),
     path('core/', include('core.urls')),
     path('front/', include('front.urls')),
+    path('', include('front.urls')),
```

Comment:
by that you create two paths to every page in front app, I think that front app should be only available on root path, and you should remove line 27

Reply:
ok ... :'(

Reply:
done